gkeonprem/gkeonprem_baremetal_cluster_metallb/main.tf

```diff
@@ -0,0 +1,155 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#[START gkeonprem_bare_metal_cluster_metallb]
+# Fill in the local variables according to your baremetal environment; you must
+# already have an admin cluster created according to the prerequisites outlined
+# in https://cloud.google.com/anthos/clusters/docs/bare-metal/latest/installing/creating-clusters/create-user-cluster-api#before_you_begin
+locals {
+  project           = "YOUR_PROJECT_ID"
```

Comment:
Please delete the `project` argument or comment it out.

We direct users to use `export GOOGLE_CLOUD_PROJECT` instead.

see https://cloud.google.com/docs/terraform/basic-commands#google_project

Reply:
Delete all occurrences:

`project                   = local.project`

Reply:
Where necessary, you can use `data "google_project"` instead

Reply:
Yes, the writer [guidelines](https://cloud.google.com/guides/authoring/terraform/include-on-cgc#include-instructions) say to include or link to the document from the docs. 

The way the CI test framework works for this repo, they spin up a new project every time, so specifying a project doesn't work. 

Can you comment out the project ID? If not, you can try adding a test.yaml file to skip testing.

Reply:
Please see the contributing guide for more information about testing.

Reply:
See also: https://cloud.google.com/guides/authoring/terraform/include-on-cgc#include-instructions

Reply:
@Shabirmean - please also update `local.project` suggestion, for PR https://github.com/terraform-google-modules/terraform-docs-samples/pull/397/files

Reply:
Cool. Thank you. I don't think we can run integration tests on these samples. They are targeted towards a specific baremetal infrastructure. So we either need to have one setup or emulate one using GCE VMs to be able to test them. Thus, I have added a skip yaml file (test.yaml).

Also regarding the project id, I have added the `data resource` with an attribute and have added comments about it. Let me know wdyt!